tests/reactive_fluid_transport.prm

```diff
@@ -0,0 +1,155 @@
+# This is a 1D test case for an insoluble fluid moving upwards in a
+# narrow channel through porous coupled two-phase flow.
```

Comment:
I think this would be more correct
```suggestion
# This is a 1D test case for a fluid moving upwards in a narrow channel 
#through porous coupled two-phase flow without reacting with the solid rock.
```